lm_eval/utils.py

```diff
@@ -175,6 +179,14 @@ def parse_infill(code, tokenizer):
                     ),
                     tokenizer,
                 )
+            # Treat eos token as a regular stop word not removing it from the output
```

Comment:
can you give an example for this behavior?

Reply:
I see, in this case shouldn't we always add the eos token to the list of stop words?

Reply:
Yes I think this is done `in generation.py`:
```
        if tokenizer.eos_token:
            task.stop_words.append(tokenizer.eos_token)
```

Reply:
I see, great catch! Will add it to the humaneval PR. This actually might explain some weird discrepancy we were observing in the MultiPL-E [PR](https://github.com/bigcode-project/bigcode-evaluation-harness/pull/44) where the model generates Java after python, hope that's it 🤞